Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name:  kepler
+description: A Helm chart for kepler
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
```

Comment:
`1.16.0` is version for helm?

Reply:
it is version generated by helm

Reply:
[templates/tests/test-connection.yaml](https://github.com/sustainable-computing-io/kepler-helm-chart/pull/1/files/0e4fd75bfef3e72ed8c824783a09178bb2886cb9#diff-e717f8763feaf34081f7215ef0388559cf3b5fdb60db18750e2ebcaabb1d4af8) was replaced with .github/workflows/integration_test.yml